GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockVirtualizationInstance.cs

```diff
@@ -174,12 +149,61 @@ public void WaitForCreateWriteBuffer()
             this.waitForCreateWriteBuffer.WaitOne();
         }
 
+        public HResult CompleteCommand(int commandId, NotificationType newNotificationMask)
+        {

```

Comment:
I think the only version of `CompleteCommand` we're calling is

`CompleteCommand(int commandId, HResult completionResult)`

Can the others just throw `NotImplementedException`?